web/tests/stream_settings_ui.test.js

```diff
@@ -212,7 +212,5 @@ run_test("redraw_left_panel", ({mock_template}) => {
     };
 
     test_filter({input: "d", subscribed_only: true}, [poland]);
-    assert.ok(!$(".stream-row-denmark").hasClass("active"));
-    assert.ok(!$(".right .settings").visible());
-    assert.ok($(".nothing-selected").visible());
+    assert.ok($(".stream-row-denmark").hasClass("active"));
```

Comment:
`.stream-row-denmark` should have `active` class irrespective of any filter changes and assert to true.
`.nothing-selected` and `.right .settings` don't make sense to assert this values since they are not being set by `redraw_left_panel` anymore.